.php_cs

```diff
@@ -0,0 +1,33 @@
+<?php
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__ . "/src");
+
+return PhpCsFixer\Config::create()
+    ->setRules(
+        [
+            '@PSR2'                               => true,
+            'array_syntax'                        => [
+                'syntax' => 'short'
+            ],
+            'class_keyword_remove'                => false,
+            'concat_space'                        => [
+                'spacing' => 'one'
+            ],
+            'combine_consecutive_unsets'          => true,
+            'general_phpdoc_annotation_remove'    => [
+                'annotations' => ['@author'],
+            ],
+            'linebreak_after_opening_tag'         => true,
+            'no_short_echo_tag'                   => true,
+            'no_useless_else'                     => true,
```

Comment:
the guy in the video warns for nesting ifs in ifs etc. like a christmas tree :) I totally agree with that. Each function should have a single responsibility, therefor unlikely to find nested if/else in our code base (hopefully ;-).

So why do we use it then?
- as stated earlier, readability
- but MORE important, by closing the if/else you show in your code that you are aware of every situation that can occur and act accordingly.